tests/unit_tests/transforms_test.py

```diff
@@ -375,6 +380,93 @@ def test_rescale_and_pad_to_size(self):
         padded_area = rescaled_padded_image[:, :, resized_image_shape[1] :]  # Bottom padding area
         self.assertTrue((padded_area == pad_val).all())
 
+    def test_keypoints_brightness_contrast(self):
```

Comment:
I think its also important to test them as part of a keypoints compose.
Perhpas we should also check that it works when used within a dataset in an additional test.

Reply:
Fixed in 0d006683020824b6b2e85be7f147019ad9057d4d

Reply:
Fixed in 0d006683020824b6b2e85be7f147019ad9057d4d